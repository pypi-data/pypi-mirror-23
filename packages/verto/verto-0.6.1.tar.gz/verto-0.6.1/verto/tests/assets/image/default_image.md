{image file-path="computer-studying-turing-test.png" alt="The computer is reading a book titled 'Talk like a human'" title="title text" alignment="left" caption="caption text" caption-link="iamaurl.com" source="whatisthis.com"}

Many humans take for granted the fact that they can easily have a conversation with another person, and choose appropriate things to say based on the conversation. The ability to do this is a form of intelligence, and for computers it isnâ€™t so easy! Many attempts have been made to design computer programs that can have a conversation with a human and sound intelligent. These computer programs are called *chatterbots*, or just *chatbots*.
