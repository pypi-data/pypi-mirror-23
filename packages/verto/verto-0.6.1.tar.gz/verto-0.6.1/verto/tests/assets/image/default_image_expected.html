<div>
<img alt="The computer is reading a book titled 'Talk like a human'" class="left-align" src="{% static 'computer-studying-turing-test.png' %}" />
<p><a href="iamaurl.com">caption text</a></p>
<p><a href="whatisthis.com">Source</a></p>
</div>
<p>Many humans take for granted the fact that they can easily have a conversation with another person, and choose appropriate things to say based on the conversation. The ability to do this is a form of intelligence, and for computers it isnâ€™t so easy! Many attempts have been made to design computer programs that can have a conversation with a human and sound intelligent. These computer programs are called <em>chatterbots</em>, or just <em>chatbots</em>.</p>
